### PR TITLE
SE-1021 Adds asgard role and other changes to deploy jenkins_admin

### PR DESCRIPTION
**Pull request by pomegranited**
Updates required to run the `edx-east/vpc_admin.yml` playbook.

* Adds asgard (`hotg`) role to configuration repo, cherry-picked from upstream
* Pulls latest `jenkins_admin` role from upstream master.
* Disables the checks and tasks which back up Jenkins to S3 -- we don't need this.

**Testing instructions**

Used to redeploy the new admin instance.  Verify that asgard is running and contains these entries in the instance type list: `t3.xlarge`, `c5.xlarge`.

**Author Notes & Concerns**

1. Changes to the asgard (`hotg`) role are submitted upstream with https://github.com/edx/configuration/pull/5348.

**Reviewer**

- [x] @swalladge

### CODE REVIEW

File: playbooks/roles/hotg/meta/main.yml [hunk 1]
```diff
@@ -21,3 +21,4 @@
 
 dependencies:
   - role: oraclejdk
+  - supervisor
```
Comment (pomegranited) [inline playbooks/roles/hotg/meta/main.yml:24]: Note: need to upstream this change.

File: playbooks/roles/hotg/defaults/main.yml [hunk 2]
```diff
@@ -51,6 +51,9 @@ HOTG_CALLBACK_URI: "{{ HOTG_URL }}/auth/signIn"
 HOTG_SUCCESS_URI: "{{ HOTG_URL }}"
 HOTG_AUTHENTICATION_PROVIDER: "githubOauthAuthenticationProvider"
 
+# Instance types configuration
+HOTG_ADDITIONAL_INSTANCE_TYPES: []
```
Comment (pomegranited) [inline playbooks/roles/hotg/defaults/main.yml:55]: Note: need to upstream this change.

File: playbooks/edx-east/vpc_admin.yml [hunk 3]
```diff
@@ -10,7 +10,6 @@
     - user
     - jenkins_admin
     - hotg
-    - alton
```
Comment (pomegranited) [inline playbooks/edx-east/vpc_admin.yml:13]: Note: Check with DevOps to find out what this role does?  We had no variables set for it, and so couldn't run it.

**Review by samuelallan72 — APPROVED** [review]
:+1:

- [X] I tested this:
  - verified that the new instance types are displayed in asgard
- [X] I read through the code
- [ ] ~I checked for accessibility issues~
- [ ] ~Includes documentation~
- [X] I made sure any change in configuration variables is reflected in the corresponding
    client's `configuration-secure` repository.

**Pull request merged**